app/version.properties

```diff
@@ -1,4 +1,4 @@
 VERSION_MAJOR=2
 VERSION_MINOR=5
 VERSION_PATCH=1
-VERSION_CODE=97
+VERSION_CODE=98

```

Comment:
version code got update on release branch (due to CI updating it before creating beta build)